src/Blazored.Typeahead/Blazored.Typeahead.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

My apologies Mark, I missed this on the first review. The RCL can stay as 2.0 otherwise anyone who's not upgraded won't be able to use the library. I think the guidance for libraries is still to target the lowest version of .NET Standard as you can to make it available to the biggest number of applications.

```suggestion
    <TargetFramework>netstandard2.0</TargetFramework>
```